src/routes/teams.rs

```diff
@@ -0,0 +1,27 @@
+use actix_web::{get, web, HttpResponse};
+use sqlx::PgPool;
+use crate::models::teams::TeamId;
+use crate::routes::ApiError;
+use crate::database::models::TeamMember;
+
+#[get("{id}/members")]
+pub async fn team_members_get(
+    info: web::Path<(TeamId,)>,
+    pool: web::Data<PgPool>,
+) -> Result<HttpResponse, ApiError> {
+    let id = info.into_inner().0;
+    let members_data = TeamMember::get_from_team(id.into(), &**pool)
+        .await
+        .map_err(|e| ApiError::DatabaseError(e.into()))?;
```

Comment:
Since `get_from_team` returns `database::models::DatabaseError`, this could just be replaced with `.await?;`.  `ApiError` implements `From<DatabaseError>`;  the other places need this because those database methods return `sqlx::Error`.

Reply:
Doesn't work, sadly